common/management/commands/project_external_updates.py

```diff
@@ -21,6 +49,89 @@ def handle(self, *args, **options):
+        # it will be in the format http[s]://trello.com/b/boardid/....
+        token = url.split('/')
+        
+        #board id will be at 4th position always
+        if len(token) >= 5:
+            board_id = token[4]
+       
+    return board_id
+
+
+def get_and_save_trello_board_actions(trello_link, created_since_date):
+    """
+    Get actions across all boards
+    :param board_ids: list of trello board ids
+    """
+    project = trello_link.link_project
+    board_id = get_trello_board_id_from_url(trello_link.link_url)
+    if board_id is not None:
+        print('Pulling actions for project {id}, Trello link: {url}'.format(
+            id=project.id, url=trello_link.link_url))
+
+        actions = get_board_actions(board_id, created_since_date)
+
+        print('Retrieved {num_actions} action(s) for board {board_id}'.format(
+            board_id=board_id, num_actions=len(actions)))
+
+        #push the actions to the model
+        push_trello_actions_to_db(project, actions)
+        latest_commit_date = datetime.now().strftime('%Y-%m-%dT%H:%M:%SZ')
+        update_if_commit_after_project_updated_time(
+            project, latest_commit_date)
+    else:
+        print('Unable to retrieve board id for trello url {}'.format(
+            trello_link.link_url))
+
+
+def push_trello_actions_to_db(project, actions):
+    from civictechprojects.models import TrelloAction
+
+    #before pushing trello actions make sure to delete the table for PII
+    if len(actions) > 0 : 
+        TrelloAction.objects.filter(action_project=project).delete()
+
+    for action in actions:
+        member = action.get("memberCreator", {})
+        data = action.get("data", {})
+
+        member_id = member.get("id")
+        member_fullname = member.get("fullName")
+        member_avatar_base_url = "" if member.get("avatarUrl", "") is None else member.get("avatarUrl", "")
+
+        board_id = data.get("board",{}).get("id")
+        action_type = action.get("type")
+        action_date = action.get("date")
+
+        id = action.get("id")
+
+        TrelloAction.create(project, 
+                            id, 
+                            member_fullname, 
+                            member_id,
+                            member_avatar_base_url,
+                            board_id, 
+                            action_type, 
+                            action_date,
+                            data)
+
+
+def get_project_trello_links():
+    """
+    Queries for Trello Board ProjectLinks. Selects one Trello link per project,
+    prioritizing links with 'link_projmanage' as the name.
+    """
+    from civictechprojects.models import ProjectLink
+    return ProjectLink.objects.filter(link_url__icontains='trello.com/b')\
+        .exclude(link_project__isnull=True)\
+        .extra(select={'is_projmanage_link': "link_name = 'link_projmanage'"})\
+        .order_by('link_project_id', '-is_projmanage_link')\
+        .distinct('link_project_id')
```

Comment:
With github, we support ingesting multiple github repos per project, so we should do the same for trello.

Reply:
Pushed an update for this.
This should now support multiple Trello board links